Tasks/DownloadBuildArtifacts/task.json

```diff
@@ -30,30 +30,55 @@
       "type": "pickList",
       "label": "Build definition",
       "defaultValue": "",
-      "required": true,
+      "required": false,
       "properties": {
         "EditableOptions": "True"
       },
       "helpMarkDown": "Select the build definition name"
     },
     {
-      "name": "version",
+      "name": "buildId",
       "type": "pickList",
       "label": "Build",
-      "defaultValue": "",
+      "defaultValue": "$(Build.BuildId)",
       "required": true,
       "properties": {
         "EditableOptions": "True"
       },
       "helpMarkDown": "Select the build number"
```

Comment:
"The build from which to download the artifacts" for helpMarkDown

Reply:
fixed,